### PR TITLE
screencap, settings, wm: add Korean translation

### PR DESCRIPTION
**Pull request by IMHOJEONG**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by IMHOJEONG** [comment]
How do I update the translation status page at https://lukwebsforge.github.io/tldri18n/?

**Comment by IMHOJEONG** [comment]
> Hello! I've noticed something unusual when checking this PR:
> 
> * The page `pages.ko/android/settings.md` is outdated, based on number of commands.
> 
> Is this intended? If so, just ignore this comment. Otherwise, please double-check the commits.

Hmm... I'm trying to read and translate the latest English version of the article, but tldr-bot keeps giving me the following error.

**Comment by IMHOJEONG** [comment]
> It will be updated automatically at 00:00 GMT. You could also check out the [tldr-maintenance-repo](https://github.com/tldr-pages/tldr-maintenance). This repo runs after every commit on main and also shows the metrics that are shown on https://lukwebsforge.github.io/tldri18n, but also checks more. Feel free to check it out!

thank you for telling me. However... page/ko/android Even though the internal files were translated and merged, they were not reflected, so I was curious.

File: pages.ko/android/settings.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`settings get {{글로벌}} {{비행기_모드_온}}`
+
+- 특정 설정 값 설정:
+
+`settings put {{시스템}} {{스크린_밝기}} {{42}}`
+
+- 특정 설정 삭제:
+
+`settings delete {{보안}} {{화면_보호기_활성화}}`
```
Comment (kbdharun) [inline pages.ko/android/settings.md:20]:
```suggestion
`settings list {{global}}`

- 특정 설정의 값 가져오기:

`settings get {{global}} {{airplane_mode_on}}`

- 특정 설정 값 설정:

`settings put {{system}} {{screen_brightness}} {{42}}`

- 특정 설정 삭제:

`settings delete {{secure}} {{screensaver_enabled}}`
```

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution. Placeholders in `settings` shouldn't be translated.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**